test/mockito_test.dart

```diff
@@ -239,6 +246,13 @@ void main() {
           throwsArgumentError);
     });
 
+    test('thenReturn throws if order', () {
```

Comment:
Test description should be more precise. "thenReturnInOrder throws on empty list input" or something like that.

Reply:
Thanks for the suggestion, I made the update, WDYT?